solr_conf_4_testing/schema.xml

```diff
@@ -144,12 +145,13 @@
     <field name="pub_country" type="text" indexed="true" stored="true" omitNorms="true"/>
     <!-- TODO: should pub_date_search be a date or a text field? -->
     <field name="pub_date_search" type="text" indexed="true" stored="true" omitNorms="true"/>
-    <!-- pub_date_sort now deprecated; replaced by pub_year_isi -->
     <field name="pub_date_sort" type="alphaSort" indexed="true" stored="true" />
     <!-- Pub Date Facet Fields -->
-    <!-- pub_year_tisim is for date slider;  pub_year_w_approx_isi/pub_year_no_approx_isi are other facets -->
+    <!-- pub_date was facet and display, now deprecated -->
+    <field name="pub_date" type="string" indexed="true" stored="true" />
```

Comment:
pub_date is the old way;  per the old comment, there are now other fields that are better for date slider and for facets.  pub_date was originally implemented to be everything in a date field, but that turned out to be not such a great plan.